### PR TITLE
docbook-xsl: support docbook.sourceforge.net URIs

### PR DESCRIPTION
**Pull request by l2dy**
#### Description
Fixes: https://trac.macports.org/ticket/55255
Fixes: https://trac.macports.org/ticket/55253

@lbschenkel Fix confirmed for NeoMutt, other dependent ports are not tested yet.

### CODE REVIEW

**Comment by l2dy** [comment]
Removed `s/[quotemeta ${version}]/1.79.1/g` since V1.79.2 is not V1.79.1 and the old version of this port references `http://docbook.sourceforge.net/release/xsl/snapshot_9899/` not `http://docbook.sourceforge.net/release/xsl/1.79.1/`.

**Comment by lbschenkel** [comment]
That was fast. :-) It looks sensible, if you verified that it fixes `neomutt` (I can suggest `opensc +doc` as well) then I have no objections to merging it.

File: textproc/docbook-xsl/Portfile [hunk 1]
```diff
@@ -58,6 +60,9 @@ destroot {
     foreach pathname ${srcdocpaths} {
         copy ${worksrcpath}/${pathname} ${destroot}${prefix}/${instdocdir}
     }
+
+    copy ${destroot}${prefix}/${instxsldir}/catalog.xml ${destroot}${prefix}/${instxsldir}/catalog.sf.xml
```
Comment (lbschenkel) [inline textproc/docbook-xsl/Portfile]: Suggestion: I would have added a short comment here to explain why this is being done, it's possible that somebody who's looking at this months from now could have a hard time understanding the reasoning.

Comment (l2dy) [reply textproc/docbook-xsl/Portfile]: Done.

**Comment by l2dy** [comment]
Tested against `neomutt` and `opensc +doc`, merging now.

P.S. Network here was down for an hour, sorry for the delay.

**Pull request merged**